dm/ctl/master/start_task.go

```diff
@@ -48,6 +49,20 @@ func startTaskFunc(cmd *cobra.Command, _ []string) error {
 		return err
 	}
 
+	// If task's target db is configured with tls certificate related content
+	// the contents of the certificate need to be read and transferred to the dm-master
+	task := config.NewTaskConfig()
+	yamlErr := task.RawDecode(string(content))
+	if yamlErr != nil {
+		return yamlErr
+	}
+	if task.TargetDB.Security != nil {
+		loadErr := task.TargetDB.Security.LoadTLSContent()
+		if loadErr != nil {
+			return loadErr
+		}
+		content = []byte(task.String())
```

Comment:
need to test `RawDecode` and `String()` won't fail the following adjust, because they may change the content

Reply:
fixed in https://github.com/pingcap/dm/pull/1910/commits/12b1757a92e30ce96a52d2ddb85f9ff869f2d8f3